src/Compilers/Core/Portable/CommandLine/AnalyzerConfigSet.cs

```diff
@@ -500,7 +500,11 @@ internal void MergeIntoGlobalConfig(AnalyzerConfig config, DiagnosticBag diagnos
                 MergeSection(config.PathToFile, config.GlobalSection, isGlobalSection: true);
                 foreach (var section in config.NamedSections)
                 {
-                    if (Path.IsPathRooted(section.Name) && !ContainsSpecialCharacters(section.Name))
+                    if (Path.IsPathRooted(section.Name) && !ContainsSpecialCharacters(section.Name)
+                        // all paths in editorconfig are converted to forward slash, 
+                        // but on windows a literal backslash is still a valid rooted path,
+                        // so we explicitly ignore any sections that start with it
+                        && !section.Name.StartsWith("\\\\", StringComparison.OrdinalIgnoreCase))
```

Comment:
Why isn't the leading backslash converted to forward slash?

Reply:
Because we're saying that it isn't a path character, but an actual backslash character

Reply:
As discussed offline, updated this check to the broader 'does this path have a forward slash' check.